orc8r/cloud/deploy/orc8r_deployer/docker/run_deployer.bash

```diff
@@ -22,7 +22,7 @@ example: ./run_deployer.bash --deploy-dir ~/orc8r_15_deployment"
 run_unit_tests()
 {
     echo "Running orcl container unit tests"
-    docker run -it \
+    docker run -i \
```

Comment:
For my understanding: why are we making this change?

Reply:
We are upgrading the ci machines from Ubuntu 16.04 with Docker 19.x to Ubuntu 20.04 with docker 20.10.
And in docker 20.x allocating a terminal while specifying a script entrypoint fails.
The circle ci job with this script still works as expected